.codecov.yml

```diff
@@ -6,15 +6,17 @@ codecov:
 coverage:
   precision: 4
   round: down
-  range: "65...90"
+  range: "70...100"
 
   status:
-    project: 
+    project:
       default:
-        threshold: 0.2 #Allow the coverage to drop by threshold%, and posting a success status.
+        target: auto
+        threshold: 0% # decline in denial coverage.
```

Comment:
If you want to set this threshold to 0%, it's better to provide another command to let one reviewer do the final check and then force merge, since we always have code lines that are hard to cover.

Reply:
I'm not quite sure whether the zero threshold here is reasonable. The coverage statistics are not always accurate, and the same modification may lead to different results in different environments.

Reply:
And as chrysan said above, code lines are not always easy to be covered, for example, some insurance codes written for extreme cases may never be generated in the test environment, which means the coverage drop would never reach 0.

Reply:
By removing some useless code and its test, the coverage will drop, but it's acceptable.